test/cmd/ephemeral/pysparkdc.json

```diff
@@ -0,0 +1,188 @@
+{
```

Comment:
Note, this is basically here because we needed to add the TEST_MODE parameter and I didn't want to litter the standard templates.